common/fw-update-helper.h

```diff
@@ -11,11 +11,11 @@ namespace rs2
     class viewer_model;
 
     int parse_product_line(std::string id);
-    std::string get_available_firmware_version(int product_line);
-    std::map<int, std::vector<uint8_t>> create_default_fw_table();
+    std::string get_available_firmware_version(int product_line, std::string PID);
+    std::map<std::pair<int, std::string>, std::vector<uint8_t>> create_default_fw_table();
```

Comment:
Please add comment what the return table is

Reply:
added